gpt_json/gpt.py

```diff
@@ -69,11 +76,11 @@ def __init__(
         # For messages that are relatively deterministic
         temperature=0.2,
         timeout=60,
-        openai_max_retries=3,
+        api_max_retries=3,
```

Comment:
nit: Let's keep backwards compatibility here. Can you add a new `api_max_retries` parameter to __init__ and flag via a logging.warn that `openai_max_retries` is deprecated?